src/BusinessLogic/DataAccess/ConnectionData/Entities/ConnectionData.php

```diff
@@ -66,8 +70,8 @@ public function toArray(): array
 
         $data['storeId'] = $this->storeId;
         $data['connectionData'] = [
-          'environment' => $this->connectionData->getEnvironment(),
-          'merchantId' => $this->connectionData->getMerchantId(),
+            'environment' => $this->connectionData->getEnvironment(),
+            'merchantId' => $this->connectionData->getMerchantId(),
```

Comment:
These extra spaces should not be there, should they?

Reply:
The code is correctly formatted now. You can see it [here](https://github.com/sequra/integration-core/blob/master/src/BusinessLogic/DataAccess/ConnectionData/Entities/ConnectionData.php). The 'environment' and 'merchantId' keys are now in the same line as 'authorizationCredentials'.